enterprise/admin/__init__.py

```diff
@@ -164,7 +164,6 @@ class EnterpriseCustomerAdmin(DjangoObjectActions, SimpleHistoryAdmin):
         'site',
         'country',
         'active',
-        'has_logo',
```

Comment:
Removing has_logo since customer records will return the platform logo URL if no branding exists